common/metrics/defs.go

```diff
@@ -1658,6 +1658,7 @@ const (
 	ReplicationTasksFetched
 	ReplicationTasksReturned
 	ReplicationDLQFailed
+	ReplicationDLQDumpCount
```

Comment:
`ReplicationTaskNackCount` as nack not necessarily means the task will be put on DLQ?